### DISCUSSION TITLE
奇忽局03号理事会：wiki理事会

### DISCUSSION THREAD

**Discussion by Lupingyue** [qihuju/qihuju.github.io#20] [General]
定义和规则有待补充……

----
以下用于抛砖引玉，by @askender, 欢迎参与编辑，稍后会在wiki里不断完善。
定义（草稿）： 构建一个基于维基和脑图的、使用下一代语义网技术且能人机协作的、内容有深度、呈现方式酷炫（好用的脑图展现形态和webvr）的无限扩展🔗的知识图谱网络和巴别图书馆

规则&玩法（草稿）（edited by @askender ）：
- 添加评论即可加入或者退出
- 每周在ch简单交流一次，采用罗伯特议事原则
  1. [Wikipedia - 罗伯特议事原则](https://zh.wikipedia.org/zh/%E7%BD%97%E4%BC%AF%E7%89%B9%E8%AE%AE%E4%BA%8B%E8%A7%84%E5%88%99)
  2. [简书 - 《罗伯特议事规则》的12条基本原则（极简版）](https://www.jianshu.com/p/bd33834dd44e)
  3. [知乎 - 罗伯特议事规则：用12原则6步法，开好一个会](https://zhuanlan.zhihu.com/p/29639932)
- 暂定大致分 策展小组(tz ming  脑图、webvr等)、协议小组（阿呆  askender 数据交换规范、下一代语义网、内容实验），后面会定义清晰的数据交换规范方便对接数据。
- 有其他想法可以编辑规则和玩法（不要直接删除文字）
- 相关具体计划产出稍后使用git-repo来保存和更新。
- 欢迎任何人参与讨论和开发

**Comment by askender** [comment]
@MRchildNEO   你有空加入么？

**Reply by Lupingyue** [reply]
> @MRchildNEO 你有空加入么？

期待 @Tz-H @MingxuanXie @MRchildNEO 加入。

**Reply by MRchildNEO** [reply]
什么时候？是直接更新吗？或者在维基上。

**Reply by Lupingyue** [reply]
> 什么时候？是直接更新吗？或者在维基上。

直接更新什么意思？你是说在这里讨论吗？维基现在有可用的模版了吗？

**Reply by askender** [reply]
@MRchildNEO    可以在http://wiki.anwensf.com/  写， 也可以在本repo的wiki里实验编写，或者在notion等实验
先完成你说的元维基（关于维基的维基），看看第一条词条是什么。 我计划的第一个词条是 社区
也说说现有各种维基的可以改进之处，以及我们要构建的维基的特别之处。

我理解的特点是：1 面向未来的维基，词条新颖，走在前沿（呃，感觉和arxiv变得更像了）  2 维基本身的语义化（md等标记可以便于信息抽取），也方便和知识图谱的结合   3   维基和行动的关联性，不仅仅是解释含义，而是描述如何构建，似乎和1有点像。

**Reply by Lupingyue** [reply]
> @MRchildNEO 可以在http://wiki.anwensf.com/ 写， 也可以在本repo的wiki里实验编写，或者在notion等实验
> 先完成你说的元维基（关于维基的维基），看看第一条词条是什么。 我计划的第一个词条是 社区
> 也说说现有各种维基的可以改进之处，以及我们要构建的维基的特别之处。
> 
> 我理解的特点是：1 面向未来的维基，词条新颖，走在前沿（呃，感觉和arxiv变得更像了） 2 维基本身的语义化（md等标记可以便于信息抽取），也方便和知识图谱的结合 3 维基和行动的关联性，不仅仅是解释含义，而是描述如何构建，似乎和1有点像。

你们先示范打样，我跟着跟进。

**Comment by MingxuanXie** [comment]
wiki以及脑图这块我目前感兴趣的是如何在webvr里空间化表现“关联”，也就是开发之前提到的[ipmm计划](https://github.com/qihuju/qihuju.github.io/discussions/5#discussioncomment-544970)

奇忽wiki本身的架构我不用参与，可能只是需要之后跟各位一起研究一下怎么对接数据。

**Reply by Lupingyue** [reply]
> wiki以及腦圖這塊我目前感興趣的是如何在webvr裡空間化表現“關聯”，也就是開發之前提到的ipmm計劃

我也很感兴趣这个方向。估计 @Tz-H 也会很有兴趣。
期待看到你的更多描述和定义。
